src/features/user/PlanetCash/index.tsx

```diff
@@ -146,26 +146,41 @@ export default function PlanetCash({
     <DashboardView
       title={t('title')}
       subtitle={
-        <p>
-          {t('description')}{' '}
-          <a
-            className="planet-links"
-            href="https://www.plant-for-the-planet.org/planetcash/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            {t('learnMoreText')}
-          </a>
-          <br />
-          <a
-            className="planet-links"
-            href="https://www.plant-for-the-planet.org/terms-and-conditions/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            {t('termsText')}
-          </a>
-        </p>
+        <div>
+          <p>
+            <Trans i18nKey="partnerSignupInfo">
+              Our Planet Cash/Bulk Codes feature is a feature specifically for
+              Plant-for-the-Planet partners. Feel free to contact us at{' '}
+              <a
+                className="planet-links"
+                href="mailto:partner@plant-for-the-planet.org"
+              >
+                partner@plant-for-the-planet.org
+              </a>{' '}
+              if you are interested.
+            </Trans>
+          </p>
+          <p>
+            {t('description')}{' '}
+            <a
+              className="planet-links"
+              href="https://www.plant-for-the-planet.org/planetcash/"
+              target="_blank"
+              rel="noreferrer"
+            >
+              {t('learnMoreText')}
+            </a>
+            <br />
+            <a
+              className="planet-links"
+              href="https://www.plant-for-the-planet.org/terms-and-conditions/"
```

Comment:
Please check if it is ok to use this link and not the pp.eco links as e.g. https://pp.eco/legal/en/terms - also not sure if your link would show German content automatically, for the footer we dynamically insert a `locale` into the URL?

Reply:
Have updated the links here - https://github.com/Plant-for-the-Planet-org/planet-webapp/commit/3c72bbc74f3b889f8a41a7c37473a9a59c4daebc